tools/integration_tests/implicitdir/implicitdir_test.go

```diff
@@ -62,20 +62,34 @@ func setUpTestDir() error {
 	return nil
 }
 
-func mountGcsfuse() error {
+func mountGcsfuse(flag string) error {
 	mountCmd := exec.Command(
 		binFile,
-		"--implicit-dirs",
 		"--debug_gcs",
 		"--debug_fs",
 		"--debug_fuse",
 		"--log-file="+logFile,
 		"--log-format=text",
+		flag,
 		*testBucket,
 		mntDir,
 	)
+
+	// Adding mount command in logFile
+	file, err := os.OpenFile(logFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		fmt.Println("Could not open logfile")
+	}
+	defer file.Close()
+
+	_, err2 := file.WriteString(mountCmd.String() + "\n")
+	if err2 != nil {
+		fmt.Println("Could not write cmd to logFile")
+	}
+
 	output, err := mountCmd.CombinedOutput()
 	if err != nil {
+		log.Println(mountCmd.String())
```

Comment:
what is the difference between log.println vs file.writeString above

Reply:
file.writeString  will write command in logFile and log.println will print mount command on terminal if mounting fails.